pymc/distributions/continuous.py

```diff
@@ -1059,10 +1059,11 @@ class Beta(UnitContinuous):
     Support   :math:`x \in (0, 1)`
     Mean      :math:`\dfrac{\alpha}{\alpha + \beta}`
     Variance  :math:`\dfrac{\alpha \beta}{(\alpha+\beta)^2(\alpha+\beta+1)}`
+    Sample size  :math:`\alpha + \beta`
```

Comment:
I am not sure about this one. It's not really a property of the distribution, but an interpretation of the parameters, assuming a specific model

Reply:
You can add this as a comment in the docstring, but I agree that this is not a property per se.

Reply:
I was hesitating as well, I will remove the reference to "sample size" without the full explanation. Should I add:

'nu : tensor_like of float, optional
        Alternative "sample size" of a Beta distribution **for the Haldane prior probability Beta(0,0)** (``nu`` > 0) .

in the parameter section?

Reply:
I think the parameter description is good enough as it is now. 

We can perhaps add a link to the relevant Wikipedia section instead of mentioning the Haldane prior?

https://en.m.wikipedia.org/wiki/Beta_distribution#Mean_and_sample_size